### PR TITLE
test: add tests for state_and_loan_entity

### PR DESCRIPTION
**Pull request by Kaminar-i**
hello @djeck1432 kindly review #260

### CODE REVIEW

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, run pylint command to fix all issues, especially adding docstring and annotation types
```
pylint apps/shared/tests/test_state_and_loan_entity.py --disable=all --enable=C0114,C0115,C0116,C0301
```

File: apps/shared/tests/test_state_and_loan_entity.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        ),
+    },
+):
+    from shared.types.base import (
+        Portfolio,
+        ExtraInfo,
+        TokenParameters,
+        InterestRateModels,
+        Prices,
+        TokenValues,
```
Comment (djeck1432) [inline apps/shared/tests/test_state_and_loan_entity.py:52]: remove unused imports:
```
        TokenValues,
        CollateralAndDebtInterestRateModels,
        CollateralAndDebtTokenParameters,
```

File: apps/shared/tests/test_state_and_loan_entity.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
+# Save this as apps/shared/tests/test_state_and_loan_entity.py
```
Comment (djeck1432) [inline apps/shared/tests/test_state_and_loan_entity.py:1]: Remove CHATGPT comment please

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by Kaminar-i** [comment]
hello @djeck1432 kindly review

**Review by djeck1432 — CHANGES_REQUESTED** [review]
@Kaminar-i You skipped this comment
Please, run pylint command to fix all issues, especially adding docstring and annotation types
```
pylint apps/shared/tests/test_state_and_loan_entity.py --disable=all --enable=C0114,C0115,C0116,C0301
```

**Comment by Kaminar-i** [comment]
@djeck1432 i did it b4 i pushed sir
Screenshot from 2024-10-30 12-36-32

**Comment by djeck1432** [comment]
you need  not only to run, but fix all these issues @Kaminar-i

**Comment by djeck1432** [comment]
@Kaminar-i If you need to explain what you should do text me in DM in telegram `djeck_vorobey1`

**Comment by djeck1432** [comment]
@Kaminar-i any updates?

**Comment by Kaminar-i** [comment]
@djeck1432 review :)

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**